api/src/main/java/io/opentelemetry/baggage/Entry.java

```diff
@@ -71,11 +66,12 @@ public static Entry create(String key, String value, EntryMetadata entryMetadata
   public abstract String getValue();
 
   /**
-   * Returns the {@link EntryMetadata} associated with this {@link Entry}.
+   * Returns the (optional) {@link EntryMetadata} associated with this {@link Entry}.
```

Comment:
```suggestion
   * Returns the {@link EntryMetadata} associated with this {@link Entry}, or {@code null} if there is no metadata.
```

Reply:
I've been preferring `EntryMetadata.EMPTY` rather than null. Did I miss a spot?

Reply:
Oh that's fine too, in that case can remove `@Nullable` I think - I just noticed the annotation but didn't look at the code

Reply:
oh yeah. I should remove that. will do!

Reply:
done